RecoHGCal/TICL/plugins/PatternRecognitionbyFastJet.cc

```diff
@@ -53,7 +53,7 @@ void PatternRecognitionbyFastJet<TILES>::buildJetAndTracksters(std::vector<Pseud
   }
 
   auto trackster_idx = result.size();
-  auto jetsSize = std::count_if(jets.begin(), jets.end(), [=](fastjet::PseudoJet jet) {
+  auto jetsSize = std::count_if(jets.begin(), jets.end(), [=, *this](fastjet::PseudoJet jet) {
```

Comment:
```suggestion
  auto jetsSize = std::count_if(jets.begin(), jets.end(), [this](fastjet::PseudoJet jet) {
```
or capture the `minNumLayerCluster_` explicitly.